release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -2893,6 +2893,18 @@ This issue affects the CPU load balancing features because these features depend
 
 * Broadcom network interface controllers in legacy Single Root I/O Virtualization (SR-IOV) do not support quality of service (QoS) and tag protocol identifier (TPID) settings for the SRIOV VLAN. This affects Broadcom BCM57414, Broadcom BCM57508, and Broadcom BCM57504. (link:https://issues.redhat.com/browse/RHEL-9881[*RHEL-9881*])
 
+* In hosted control planes for {product-title}, the following Operators and components are not tested (link:https://issues.redhat.com/browse/OCPSTRAT-605[*OCPSTRAT-605*]):
+
+** Performance Addon Operator
+** Red Hat OpenShift sandboxed containers
```

Comment:
There's an attribute for this too: {sandboxed-containers-first}.
However, it resolves in _OpenShift sandboxed containers_ and not _Red Hat OpenShift sandboxed containers_